node/src/database/config_dumper.rs

```diff
@@ -362,16 +385,15 @@ mod tests {
         let data_dir = ensure_node_home_directory_exists(
             "config_dumper",
             "dump_config_dumps_existing_database_with_correct_password",
-        )
-        .join("MASQ")
-        .join(TEST_DEFAULT_CHAIN.rec().literal_identifier);
+        );
+        let chain_specific_dir = add_chain_specific_directories(Chain::PolyMainnet, &data_dir);
```

Comment:
Maybe `data_dir` is enough but `chain_specific_data_dir` would be better at minimum.